### ISSUE TITLE
Deprecated pwa dependency

### ISSUE THREAD

**Issue by Armaniimus** [BayBreezy/nuxt3-vuetify3-starter#17]
The pwa dependency "@kevinmarrec/nuxt-pwa" is deprecated
this was posted by the author on his [github page](https://github.com/kevinmarrec/nuxt-pwa-module?tab=readme-ov-file#readme)

The author advises to use ["@vite-pwa/nuxt"](https://vite-pwa-org.netlify.app/frameworks/nuxt.html)
as its in a state that it performes the required functionality natively

**Comment by BayBreezy** [comment]
Hey @Armaniimus ,

The PWA module is indeed deprecated. The funny part is that it was not added to be a part of the template( I thought users would just remove it in favour of vite's PWA)

I like the module because it's simple and easy to use. 

Vite's solution is overly complicated. I tried it a few times and did't like it. It felt like I had to do too much.

I will do one of two things though... Replace kevin's solution with Vite's... or just remove it 🤷🏽‍♂️
